build/includes/release.mk

```diff
@@ -96,5 +96,4 @@ do-release: $(ensure-build-image)
 	@echo "Now go make the $(RELEASE_VERSION) release on Github!"
 
 build-release:
-	docker run --rm $(common_mounts) -w $(workdir_path) $(build_tag) \
-		gcloud builds submit . --substitutions _VERSION=$(base_version) --config=./build/release/cloudbuild.yaml $(ARGS)
+	cd $(agones_path) && gcloud builds submit . --substitutions _VERSION=$(base_version) --config=./build/release/cloudbuild.yaml $(ARGS)
```

Comment:
This shouldn't appear in this PR - it looks like you might've done a merge into this branch again?